Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:20
+
+# Expose port for Soul server
+EXPOSE 8000
+
+COPY package.json package-lock.json ./
+
+RUN npm install
+
+RUN npm install -g soul-cli
+
+# Set the database file path as an environment variable
+ENV DB_FILE=/data/foobar.db
```

Comment:
Are we duplicating the DB_FILE variable in the `.env` file here?  Is it one or other, or do we need both?